src/hip_stream.cpp

```diff
@@ -254,14 +254,137 @@ hipError_t hipStreamGetPriority(hipStream_t stream, int* priority) {
 
 
 //---
+hipError_t setCallbackPacket(hsa_queue_t* queue,
```

Comment:
If you look at the usage of this internal function it would never return anything other than hipSuccess. Moreover we don't check the return value of this function anywhere. So it should be fine to change return type to void.